components/button.css

```diff
@@ -0,0 +1,74 @@
+
+.btn {
+    margin: 1rem;
+    height: fit-content;
+    width: fit-content;
+    border-radius: 2em;
```

Comment:
maintain consistency in units

Reply:
ok i wii do

Reply:
some typo was there

Reply:
i done that but not commited i will commit